ProjectSolenodon/Assets/Resources/LevelSaves/level-1.json

```diff
@@ -0,0 +1 @@
+{"numRegisterCards":2,"numStackCards":2,"numQueueCards":2,"numHeapCards":2}
```

Comment:
Are we still using JSON files?

If it's not going to be used it should be removed.